spec/integration/google_tag_manager_integration_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'support/capybara_app_helper'
+
+RSpec.describe "Google Tag Manager Integration" do
+  before do
+    setup_app(action: :google_tag_manager) do |tracker|
+      tracker.handler :google_tag_manager, { tracker: 'GTM-ABCDEF' }
+    end
+    visit '/'
+  end
+
+  subject { page }
+
+  it "embeds the script tag with tracking event from the controller action" do
+    expect(page.find("head")).to have_content('GTM-ABCDEF')
```

Comment:
That's absolutely right, it's even right there in the quickstart guide https://developers.google.com/tag-manager/quickstart
Thanks for catching this.